prac_05/word_occurences.py

```diff
@@ -0,0 +1,21 @@
+"""
+Emma Naumann
+CP1404 Practical 5
+
```

Comment:
What does this program do? Doc strings are nice when they say what the program does and is a helpful reminder to yourself when programming as to what your goal is

Reply:
👍